tests/unit/capi/GEOSIntersectsTest.cpp

```diff
@@ -150,5 +150,38 @@ void object::test<6>
     ensure_equals(r1, 1);
 }
 
+// https://trac.osgeo.org/geos/ticket/1110
+template<>
+template<>
+void object::test<7>
+()
+{
+    std::string wkb1 = "0103000080010000000C000000D76CFA7C7CF3FFC1B982482636CDCBC15EC31B0E65B6C1C12E877662F4E700C2103B2D82C78DCFC1A29779FE5D09E1C123CB6770DEF3FFC1FCDADA768882CBC17A8FA0B9C080CBC1482374FCF95200C26C3E6539D010EB41502415824E2BDCC1F846A740265CFCC1849036017A69EC41C0AE6BA9836DB6C1B03E5F6E25BBEFC16CB0BE5B0C05E54156D5BAAB3988FDC1887692F5A0B5E1C14E5950599AB8C6413600EDEF3251F3C1483B4D575A32FF417C70666B8C74EB410A04298CBF8AE7C1AEEB2DC41939E1C16D2B823AAC97C441FCE7F04C7BBDE0C184D82356622AEA4161A19916C828F5C1414C7E9080C9FAC12A178C59F5F0FEC1668C89B9F61CFCC1E8964B49B6FEEC41D76CFA7C7CF3FFC1B982482636CDCBC15EC31B0E65B6C1C1";
```

Comment:
Would be nicer to use WKT here, so it's obvious that the input has en EMPTY element.

Reply:
The following test uses WKT for clarity. This test uses the geometry from the ticket.